libs/java/client_common/src/main/java/com/yahoo/athenz/common/utils/SignUtils.java

```diff
@@ -171,6 +173,8 @@ private static Struct asStruct(Role role) {
         appendObject(struct, ATTR_SELF_SERVE, role.getSelfServe());
         appendObject(struct, ATTR_SERVICE_EXPIRY_DAYS, role.getServiceExpiryDays());
         appendObject(struct, ATTR_SERVICE_REVIEW_DAYS, role.getServiceReviewDays());
+        appendObject(struct, ATTR_GROUP_EXPIRY_DAYS, role.getGroupExpiryDays());
```

Comment:
the attributes in the signed document are in sorted attribute order. so this not the right place for these values.

Reply:
Fixed